services/registrations/handler.js

```diff
@@ -227,101 +229,17 @@ export async function sendEmail(user, existingEvent, userStatus, id, emailType =
-      tempId = "d-8d272b62693e40c6b469a365f7c04443";
-      status = "waitlisted";
+    const EmailService = new SESEmailService(awsConfig);
+    if (!user.isPartner) {
+      await EmailService.sendDynamicQR(userEmail, userName, existingEvent.ename, existingEvent.id, existingEvent.year, userStatus, emailType);
     }
-    // for application status email
-    if (userStatus === "reviewing") {
-      tempId = "d-8d272b62693e40c6b469a365f7c04443";
-      status = "reviewing";
-    }
-
-    // send the email for the registration status (qr code)
-    // for the calendar invite code, go to helpers.js
-    const dynamicMsg = {
-      to: userEmail,
-      from: {
-        email: "info@ubcbiztech.com",
-        name: "UBC BizTech"
-      },
-      templateId: tempId,
-      dynamic_template_data: {
-        subject:
-          emailType === "application" ?
-            "BizTech " + existingEvent.ename + " Event Application Status"
-            : "BizTech " + existingEvent.ename + " Event Registration Confirmation",
-        name: userName,
-        registrationStatus: status,
-        eventName: existingEvent.ename,
-        eventID: id
-      }
-    };
-
-    // objects below are for the calendar invite!
-    const startDate = new Date(existingEvent.startDate);
-
-    // format the event date from startDate like "October 19 9:00 AM PDT" (ensure it's PST/PDT) then append location
-    // check if PDT or PST
-    const timeZone = startDate.getTimezoneOffset() === 420 ? "PDT" : "PST";
-    const eventDate =
-      startDate.toLocaleString("en-US", {
-        timeZone: "America/Los_Angeles",
-        month: "long",
-        day: "numeric",
-        hour: "numeric",
-        minute: "numeric",
-        hour12: true
-      }) +
-      " " +
-      timeZone +
-      " — " +
-      existingEvent.elocation;
-
-    // format event day like "October 19"
-    const eventDay = startDate.toLocaleString("en-US", {
-      timeZone: "America/Los_Angeles",
-      month: "long",
-      day: "numeric"
-    });
-
-    const dynamicCalendarMsg = {
-      templateId: tempCalendarId,
-      dynamic_template_data: {
-        EVENT_NAME: existingEvent.ename,
-        BIZTECH_HOMEPAGE: "https://app.ubcbiztech.com",
-        CURRENT_YEAR: existingEvent.year,
-        FIRST_NAME: user.fname,
-        BANNER_URL: existingEvent.imageUrl,
-        EVENT_DATE: eventDate,
-        EVENT_DAY: eventDay,
-        EVENT_URL: "https://app.ubcbiztech.com/events",
-        SUBJECT: `[BizTech Confirmation] ${existingEvent.ename} on ${eventDay}`
-      }
-    };
-
-    await registrationHelpers.sendDynamicQR(dynamicMsg);
-    if (emailType !== "application" && userStatus === "registered" && tempCalendarId)
-      await registrationHelpers.sendCalendarInvite(
-        existingEvent,
-        user,
-        dynamicCalendarMsg
-      );
+    if (emailType !== "application" && userStatus === "registered")
+      await EmailService.sendCalendarInvite(existingEvent, user);
```

Comment:
The intended behaviour here is that Partners should not receive a .ics calendar invite. This is because Partners team wants to manually send them calendar invites with personalized blocks of time when they are expected to be present.

The implementation here i think is sending partners the cal invite, but not the QR checkin code, which is backwards from the intended functionality. 
However, the QR checkin code email that we use is pretty bare bones, so the future here is to use this new AWS SES class to create a partner-specific email.

For now in this PR, let's just make remove the isPartner flag on line 238, and add a TODO comment for documentation on making a partner-specific email without the .ics in the future. I'm making another PR rn to just disable all partner emails, and we can revisit the partner email when time comes.